src/img/tester.py

```diff
@@ -588,7 +589,7 @@ def run_tests(self):
                 else:
                     print "trying to get any test results"
                     self.commands.scpfrom("/tmp/results/*", self.results_dir)
-                    self.commands.ssh(['rm', '-rf', '/tmp/results/*'])
+                    self.commands.ssh(['sh', '-c', 'rm -rf /tmp/results/*'])
```

Comment:
Notice how the original arguments differ from the invocation of run() couple of lines above. Now ssh() and run() can be used equally just like popen with list of string.

Reply:
This confuses me. We end up with
 
   sh -c sh -c "'rm -rf /tmp/results/*'"

do we need a double sh -c to undo a pipes.quote level?